celerdata/metadata.csv

```diff
@@ -0,0 +1,543 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
```

Comment:
Please reformat the description as complete sentences, for example: 

` Job statistics.`
`The maximum journal ID of this frontend.`

Also noting that there are missing metric descriptions in here.

Reply:
‌‌The reformatting issue has been resolved.

Reply:
Hey, it looks like a lot of these have 11 columns instead of 10 and isn't a valid CSV file, could you please fix this? 
(Validations have been improved, so future commits on this PR should catch those and report errors back to you)

Reply:
Yes, have Fixed it